app/server/utils/ge_migrations/rds_migrate.py

```diff
@@ -0,0 +1,376 @@
+
+            self.migrate_balances(ge_address_to_transfer_account)
+
+
+    def insert_user(self, ge_user):
+
+        phone_number = None if 'DELETED' in ge_user['phone'] else ge_user['phone']
+
+        if not phone_number:
+            return
+
+        processed_phone = proccess_phone_number(phone_number)
+        existing_user = User.query.filter_by(phone=processed_phone).execution_options(show_all=True).first()
+        if existing_user:
+            print(f'User already exists for phone {processed_phone}')
+            return
+
+        if ge_user['admin_id']:
+            return
+
+        if ge_user['business_type'] is not None:
+            transfer_usage = TransferUsage.find_or_create(ge_user['business_type'])
+            business_usage_id = transfer_usage.id
+        else:
+            business_usage_id = None
+
+        if business_usage_id:
+            business_usage = TransferUsage.query.get(business_usage_id)
+        else:
+            business_usage = None
+
+        organsation = db.session.query(Organisation).get(self.sempo_organisation_id)
+
+        try:
+            sempo_user = create_transfer_account_user(
+                first_name=ge_user['first_name'],
+                last_name=ge_user['last_name'],
+                organisation=organsation,
+                phone=phone_number,
+                preferred_language=ge_user['preferred_language'],
+                location=ge_user['location'],
+                business_usage=business_usage
+            )
+
+            sempo_user.is_activated = ge_user['status'] == 'Active'  # Is this the correct way to find this out?
+            sempo_user.is_disabled = False
+            sempo_user.is_phone_verified = True
+            sempo_user.is_self_sign_up = False
+            sempo_user.terms_accepted = False
+            sempo_user.created = ge_user['created_at']
+            sempo_user.custom_attributes = self.create_custom_attributes(ge_user)
+
+            if ge_user['token_agents.id'] is not None:
+                sempo_user.set_held_role('TOKEN_AGENT', 'grassroots_token_agent')
+            else:
+                # Is this the correct way to find this out or can a benificiary also be a token agent 
+                # Or is there some field where you can find this out?
+                sempo_user.set_held_role('BENEFICIARY', 'beneficiary')
+
+            if ge_user['group_accounts.id'] is not None:
+                sempo_user.set_held_role('GROUP_ACCOUNT', 'grassroots_group_account')
+
+            db.session.flush()
+
+            return sempo_user
+
+        except (IntegrityError, InvalidRequestError) as e:
+            print(e)
+            db.session().rollback()
+
+
+
+    def create_custom_attributes(self, ge_user):
+
+        wanted_custom_attributes = [
+            ('id', 'GE_id'),
+            ('wallet_address', 'GE_wallet_address'),
+            ('bio', 'bio'),
+            ('national_id_number', 'national_id_number'),
```

Comment:
@tristanhcole is this fine to just save in custom attributes, or is there somewhere more sensible to store it?

Reply:
any identity data should be stored on KYC object imo

Reply:
also, is_phone_verified should default to true for USSD users (assuming the user phone === ussd session phone)

Reply:
`is_phone_verified` is already set to true for USSD users migrating (see line 162 above), but I'll create a side effect of setting pins that it also confirms ur phone as verified....